src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,65 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService mnfService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        DriverService drvService = (DriverService)
+                injector.getInstance(DriverService.class);
+
+        Manufacturer manufacturer1 = mnfService.get(1L);
```

Comment:
It's bad practice to use numbers in variable names. And why we get some Manufacturer from db while we haven't add some there before?